publication-model/src/main/java/no/unit/nva/model/associatedartifacts/file/PendingInternalFile.java

```diff
@@ -89,6 +82,6 @@ public InternalFile approve() {
 
     @Override
     public boolean isNotApprovable() {
-        return isNull(getLicense()) || isNull(getPublisherVersion());
+        return isNull(getLicense());
```

Comment:
We have publication types that does not require files to have version, for example `Utstilling`
Checking license only.